source/mainnet/net/resources/release-notes-mainnet.rst

```diff
@@ -11,6 +11,21 @@ Release notes - Mainnet
 
    To subscribe to updates on the Mainnet status page click **Subscribe** to get all updates or click **Get updates** to choose to get all updates or only updates for specific products.
 
+May 25, 2023
+
+|bw| 1.0.5
+-------------------------
+
+|bw| 1.0.5 contains fixes for the following issues:
+
+- The About page link to the terms and conditions pointed to the wrong URL. It now uses the value retrieved from the wallet proxy, or the correct default to the unified terms and conditions page.
+
+- Fixed an empty recovery displaying an error instead of informing the user that nothing was found.
+
+- Fixed an issue where the transaction list view would show the Request CCD button while loading the initial batch of transactions.
+
+- Added a missing translation for the Request CCD button.
+
```

Comment:
We are also including:

-   Added text that a transaction has been submitted.
-   Messages when confirming baker/delegation transactions no longer appears after the transaction has been submitted.
-   `deployModule` transactions now display the hash instead of the entire module source.
-   `deployModule` transactions no longer cause the wallet to crash, when supplied with `sendTransaction`.

Though maybe the deployModule lines could just be:
-   `deployModule` transactions are now supported in the `sendTransaction` endpoint of the wallet-api.

or something similar :thinking: